_posts/2020-08-06-kubernetes-clients-comparison.md

```diff
@@ -0,0 +1,213 @@
+
+When first starting Lens, it immediately feel very easy to use.
+Adding a cluster can be done by hitting the `+` and selecting a cluster from the dropdown.
+Lens leverages the contents of the Kubeconfig it finds on the system to discover and authenticate with clusters.
+This way no additional magic is needed to get started.
+
+After connecting to a cluster, you're dropped into the cluster overview (see screenshot).
+This view provides you with an easy overview of the resources within the cluster and (super useful) provides a list of the last seen error events in the cluster.
+
+Navigating to the list of pods provides an overview of all pods in the cluster.
+By clicking on a pod you're provided with the details of that pod (`kubectl describe`).
+From here, you can directly dive into the pod logs, shell into the pod, make edits or remove the pod from the cluster.
+
+Similar support is available for most common resources within the cluster: Statefulsets, deployment, configmaps, secrets, ...
+The workflow is always the following: open the type in the sidebar on the left, click on an object to get details.
+From that details view, certain actions can be performed on the object.
+
+As with most of the tools in this comparison, Lens is quite feature-rich.
+Most common types are support and common actions are available.
+But Lens has another trick up it's sleeve which makes it different from the other tools: Metrics/Prometheus integration.
+
+The integration relies on a Prometheus instance being installed in the cluster that exposes the supported metrics.
+You can opt for Lens to install Prometheus (and other required components) for you, but in real scenarios you either don't have those rights or you'll already have a Prometheus installed in the cluster.
+In those cases, you can just [add some configuration](https://github.com/lensapp/lens/blob/master/troubleshooting/custom-prometheus.md){:target="_blank" rel="noopener noreferrer"} to that instance and point the Lens app to that Prometheus instance.
+It will use port-forwarding under the hood, so no need to expose the Prometheus instance to the outside.
+The integration works nicely and instantly provides some metrics about your cluster and deployed components.
+This provides good insights for developers to figure out their resource consumption without leaving their Kubernetes client.
+The charts and data seem to be very rudimentary, but improvement are expected to arrive over time.
+
+As most of the client described in this post, Lens app is an open-source project.
+There seems to be a group / company behind the development, but no paid / supported version is available at this time.
+There is very active development happing on the app and releases are about one month apart, so bugs and new features should be available regularly.
```

Comment:
happening*

Reply:
"so bugs and new features should be available regularly". die bugs ertussen vind ik raar, of ligt dat aan mij? :D